backend/audit/intakelib/checks/check_gsa_migration_keyword.py

```diff
@@ -32,6 +32,9 @@ def check_for_gsa_migration_keyword(ir):
         "contains_chart_or_table",
         "is_minimis_rate_used",
         "compliance_requirement",
+        "auditee_zip",
```

Comment:
Here, we only track workbook fields (i.e., named ranges). `auditee_zip` and `auditor_zip` are part of `general_information` section.  This should go [here ](https://github.com/GSA-TTS/FAC/blob/main/backend/audit/validators.py#L247)